syntax="proto2";

import "aap_protobuf/shared/MessageStatus.proto";

package aap_protobuf.service.bluetooth.message;

message BluetoothPairingResponse
{
    required shared.MessageStatus status = 1;
    required bool already_paired = 2;
}
